### ISSUE TITLE
Due and Reminder short syntax

### ISSUE THREAD

**Issue by hiddenman** [johannesjo/super-productivity#1476]
## Problem Statement
When i add a task i want to avoid mouse clicking and specify task due date and its reminder  in a short syntax like it's done in other software
## :grey_question: Possible Solution
Add support for a short syntax like @11:30 will be "due date: today at 11:30 + default reminder from the settings".  
## :heavy_plus_sign: Additional context
I've checked this closed issue https://github.com/johannesjo/super-productivity/issues/487 and still can't understand if it's already supported. According to the https://github.com/johannesjo/super-productivity/blob/master/src/app/features/tasks/short-syntax.util.ts the due date parsing is still disabled.

There are plenty of small libraries that can help with this.

**Comment by johannesjo** [comment]
Thanks for opening this up. Would be nice to have this.

**Comment by asmagen** [comment]
Would be great to have it.
(I don't see upvote button)

**Comment by rogerbordini** [comment]
Yes, please this feature would be great! The vast majority of to-dos apps nowadays have this simple function and it makes it very easy to enter start dates, deadlines and reminders, both for PC/desktop and Android.
Thanks for the great app and keep up the excellent work!
